lib/src/components/NxTabPanel/NxTabPanel.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { ActiveTabContext } from '../NxTabs/NxTabs';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxTabPanel = function NxTabPanelElement(props: Props) {
+  // Use React.useContext instead of importing useContext for jest to mock the value in the test
+  const activeTab = React.useContext(ActiveTabContext);
+  const {labelledBy, className, ...attrs} = props;
+
+  if (activeTab !== labelledBy) {
+    return null;
+  }
+
+  return (
+    <div role="tabpanel"
+         aria-expanded="true"
+         aria-labelledby={labelledBy}
```

Comment:
This doesn't seem to do what we want and in fact seems to have a negative consequence: it causes the screen reader to only read out the text of the tab label when the tab panel is selected, and **not** to read out the text contained in the actual tab panel (noticeable if you change the examples to have different contents in the panels than the labels).

Reply:
I based this implementation on the text from the w3c. A direct link doesn't work of course but if you go to https://www.w3.org/TR/wai-aria-1.1/ and search for tab (role) you'll find the relevant documentation. Looks like I may have mis-read the docs the first time though. Looks like the tab panel must be owned by the tab which means I should be using aria-owns on the tab to reference the tab panel by id or we need to make the tab panel a child of the tab element. I need to rethink the interface here a bit.

Reply:
Looking at it a bit more, I think using `aria-labelledby` is correct (or at least there are many react tab implementations that use `aria-labelledby`). Material-ui for example uses `aria-labelledby` and `aria-controls`. I'll add `aria-controls` as well and make setting the ids unnecessary.

Reply:
> I based this implementation on the text from the w3c. A direct link doesn't work of course but if you go to https://www.w3.org/TR/wai-aria-1.1/ and search for tab (role) you'll find the relevant documentation. Looks like I may have mis-read the docs the first time though. Looks like the tab panel must be owned by the tab which means I should be using aria-owns on the tab to reference the tab panel by id or we need to make the tab panel a child of the tab element. I need to rethink the interface here a bit.

I agree what you've done is correct by the spec.  I've also been able to get ChromeVox to read the tab content, though it requires use of their special keyboard shortcuts.